src/openzaak/components/catalogi/tests/admin/test_admin_publish.py

```diff
@@ -87,6 +95,12 @@ def test_publish_zaaktype(self, m):
         )
         self.assertIsNotNone(publish_button)
 
+        # Verify notification is sent
+        called_urls = [item.url for item in m.request_history]
+        self.assertIn(
+            "https://notificaties-api.vng.cloud/api/v1/notificaties", called_urls
+        )
```

Comment:
For testing the notification is send when a new version is created or is published I just added to the current tests but if you prefer a new unit test I can do that. :)

Reply:
it's fine, good integration test